src/main.cpp

```diff
@@ -340,6 +340,8 @@ void Main::add_supported_options()
         , "0 = normal run, 1 = preprocess and dump, 2 = read back dump and solution to produce final solution")
     ("polar", po::value<string>()->default_value("auto")
         , "{true,false,rnd,auto} Selects polarity mode. 'true' -> selects only positive polarity when branching. 'false' -> selects only negative polarity when branching. 'auto' -> selects last polarity used (also called 'caching')")
+    ("chronophase", po::value(&conf.chronophase)->default_value(conf.chronophase)
+        , "Phase to choose during chrono BT. 0 = default, 1 = lsids, 2 = random")
```

Comment:
Ah, man, what is random? Why. No. I'll remove this, this is not meaningful.

Reply:
Actually "2" is not even handled. Please remove this random thing, it's not even there.